### DISCUSSION TITLE
RTL language support?

### DISCUSSION THREAD

**Discussion by brightloudnoise** [segmentio/evergreen#1348] [Q&A]
Does the library support RTL languages? I haven't found any documentation on how to handle that with Evergreen.

**Comment by mshwery** [comment]
I don't believe Evergreen does anything specific to accommodate or change its behavior for RTL languages, but there's nothing precluding you from using them. If you spot anything missing or broken with RTL languages, feel free to open an issue!

**Reply by AhmedElTabarani** [reply]
> I don't believe Evergreen does anything specific to accommodate or change its behavior for RTL languages, but there's nothing precluding you from using them. If you spot anything missing or broken with RTL languages, feel free to open an issue!

When setting the RTL property for HTML, I expected the padding and margin values to be flipped from left to right automatic. see this issue https://github.com/bndw/wifi-card/issues/250
